.github/workflows/php-js-e2e-tests.yml

```
@@ -0,0 +1,123 @@
+name: PHP, JavaScript, and E2E tests
+
+on:
+  push:
+    branches: [trunk]
+  pull_request:
+    branches: [trunk]
+
+jobs:
+  E2E_Tests:
+    name: End to end tests
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
```

This seems appropriate, since this is an "omnibus" of various tests.

---

Hmm .. in my testing, I only saw the first test fail, so maybe better to fail fast.

---

Yep, this only applied to matrices, and we're not using these. The new way of splitting everything into its own job will show us individual failure points.